### DISCUSSION TITLE
How to do internationalised templates?

### DISCUSSION THREAD

**Discussion by hexwit** [foundation/foundation-emails#1134] [Q&A]
Hi, 
I have checked the docs but don't see any tips about the topic regarding 
internationalisation and localisation.

Is it supposed to have separate email template for each language? 
Or what technique do you use to have multi-language templates?

Thanks

**Comment by elcaptain** [comment]
Hi, where do you store your translations? my workflow was to create my english template first and then duplicate this file and paste in the translations into the html. repeating this for every language. this became very unsustainable very quickly as our language count grew.
now we switched to use placeholder variables for each block of copy that get populated with the correct language when sending out the emails, so we only need one template file.
